src/main/scala/io/github/pauljamescleary/petstore/domain/users/Role.scala

```diff
@@ -0,0 +1,19 @@
+package io.github.pauljamescleary.petstore.domain.users
+
+import cats._
+import cats.implicits._
+import tsec.authorization.{AuthGroup, SimpleAuthEnum}
+
+sealed case class Role(roleRepr: String)
+
+object Role extends SimpleAuthEnum[Role, String] {
+
+  val Admin: Role = Role("Admin")
+  val Customer: Role = Role("Customer")
+
+  override val values: AuthGroup[Role] = AuthGroup(Admin, Customer)
+
+  override def getRepr(t: Role): String = t.roleRepr
+
+  implicit val eqRole: Eq[Role] = Eq.fromUniversalEquals[Role]
+}
```

Comment:
Add new line here and in more places

Reply:
I checked if this could be added as a scalafmt rule but seems not to be implemented https://scalameta.org/scalafmt/docs/configuration.html#newlines